resources/lang/de/messages.php

```diff
@@ -306,11 +306,11 @@
     'Start Date'                => 'Startdatum',
     'modem_restart_error'       => 'Das Modem konnte nicht neugestartet werden! (offline?)',
     'Contact Persons'           => 'Antennengemeinschaft/Kontakt',
-    'modem_restart_success_cmts' => 'Das Modem wurde erfolgreich über das CMTS neugestartet',
+    'modem_restart_success_netgw' => 'Das Modem wurde erfolgreich über das NetGw neugestartet',
     'Accounting Text (optional)'=> 'Verwendungszweck (optional)',
     'Cost Center (optional)'    => 'Kostenstelle (optional)',
     'Credit Amount'             => 'Gutschrift - Betrag',
-    'modem_restart_success_direct' => 'Das Modem wurde erfolgreich direkt über SNMP neugestartet',
+    'modem_restart_success_direct' => 'Das Modem wurde erfolgreich direkt über SNMP/TR069',
```

Comment:
```suggestion
    'modem_restart_success_direct' => 'Das Modem wurde erfolgreich direkt über SNMP/TR069 neu gestartet',
```
we could geve a variable to be specific whether it was restarted via SNMP or TR069